packages/terra-cell-grid/docs/README.md

```diff
@@ -0,0 +1,17 @@
+# Terra Cell Grid
+
+The terra-cell-grid is a structural row based layout for aligning columns with 100% of it's parents width.
```

Comment:
it's parents width -> its parent's width

Reply:
Updated.